Sources/Common/RouteType.swift

```diff
@@ -31,7 +31,7 @@ import Alamofire
 public protocol RouteType: URLRequestConvertible {
     
     /// HTTP method
-    var method: Alamofire.HTTPMethod { get }
+    var method: HTTPMethod { get }
```

Comment:
something that we end up doing in each new project is defining `GetRouteType`, `PostRouteType`, etc. Maybe would be nice to add these protocols natively in here.

What do you think?